### ISSUE TITLE
Answer these questions

### ISSUE THREAD

**Issue by jxnl** [jxnl/TwitterAlcoholDetection#41]
From the June 2015 data set -- please clarify (as I don't have a record of this): are these data from the 1% API, and you mentioned North America - so is the data limited to there? How many Tweets in total?

Can you please let me know how many Tweets end up being in each of the categories after classification, and also the performance of classification at each level of the hierarchy? 

One thing I'm thinking about is if these numbers are particularly low for alcohol vs. other behaviors, or really, is it possible to somehow get a sense of the probability someone Tweets about their drinking behavior (casual, reflecting or recalling).

**Comment by jxnl** [comment]
# Performance

![](https://cloud.githubusercontent.com/assets/4852235/11228016/dd903e82-8d57-11e5-89cb-58ccd3aea73d.png)

**Comment by jxnl** [comment]
# Limits of the data

You'd have to ask our source to see if it was the 1% or the 10% source. We have 50% filtered tweets and 50% all tweets. However all of our visualizations are only using the "all tweets" set, we also made sure that all tweets we kept had location data.
- **804000** Tweets
- **228405** Users

**Comment by jxnl** [comment]
# Frequency in First Person:

| Cut off probability | Count | Proportion |
| --- | --- | --- |
| 0.5 | 73306 | 0.0913050572385 |
| 0.6 | 46435 | 0.0578363344456 |
| 0.7 | 21388 | 0.0266394642214 |
| 0.8 | 5359 | 0.00667481245384 |
| 0.85 | 1973 | 0.0024574370165 |

**Comment by jxnl** [comment]
## Classes by DayofWeek

<img width="1011" alt="screen shot 2016-01-17 at 8 17 57 pm" src="https://cloud.githubusercontent.com/assets/4852235/12383160/8b6ead2a-bd57-11e5-9d09-54f3649eb377.png">

## Classes by HourofDay

<img width="1012" alt="screen shot 2016-01-17 at 8 18 27 pm" src="https://cloud.githubusercontent.com/assets/4852235/12383165/9cb877f0-bd57-11e5-8b4e-967826c27e1b.png">

## Classes by HourofWeek

<img width="1008" alt="screen shot 2016-01-17 at 8 20 11 pm" src="https://cloud.githubusercontent.com/assets/4852235/12383178/c9aa0c10-bd57-11e5-8616-9422af853e41.png">

**Comment by jxnl** [comment]
## Classes by DayofWeek

<img width="1009" alt="screen shot 2016-01-17 at 8 27 12 pm" src="https://cloud.githubusercontent.com/assets/4852235/12383282/f3d74df8-bd58-11e5-920d-2990eaa92f2d.png">

## Classes by HourofDay

<img width="1011" alt="screen shot 2016-01-17 at 8 27 00 pm" src="https://cloud.githubusercontent.com/assets/4852235/12383283/f4d5b60e-bd58-11e5-9b59-4cb3ac184423.png">

## Classes by HourofWeekDay

<img width="1012" alt="screen shot 2016-01-17 at 8 27 20 pm" src="https://cloud.githubusercontent.com/assets/4852235/12383284/fd8da0fe-bd58-11e5-99ec-98da70ea1564.png">

**Comment by jxnl** [comment]
# Dominant classes by day

| Day | Dominant Class |
| --- | --- |
| monday | reflection |
| tuesday | looking, reflection |
| wednesday | looking, reflection |
| thursday | looking |
| friday | looking |
| saturday | casual |
| sunday | casual |

**Comment by jxnl** [comment]
# Weekend vs Weekday

<img width="1008" alt="screen shot 2016-01-18 at 2 48 24 pm" src="https://cloud.githubusercontent.com/assets/4852235/12404698/9986f57a-bdf2-11e5-8fc8-717d99944413.png">
<img width="952" alt="screen shot 2016-01-18 at 2 48 11 pm" src="https://cloud.githubusercontent.com/assets/4852235/12404699/99aa65be-bdf2-11e5-8c98-c27c2b656188.png">

**Comment by jxnl** [comment]
# Weekend vs Weekday

![unknown](https://cloud.githubusercontent.com/assets/4852235/12404773/6ed7c6c8-bdf3-11e5-9cf1-9971ff91ee9a.png)
![unknown-1](https://cloud.githubusercontent.com/assets/4852235/12404774/6ed988aa-bdf3-11e5-8165-db55e6ad8010.png)

**Comment by jxnl** [comment]
# Separated Rates

![unknown-2](https://cloud.githubusercontent.com/assets/4852235/12404868/44219db8-bdf4-11e5-92ef-88958a2f247d.png)

**Comment by jxnl** [comment]
@rumichunara

**Comment by jxnl** [comment]
# Training / Testing data

we use a 33% split for all data, it may be in our best interest to throw in and get much more training data for the paper.

| alcohol | non_alcohol |
| --- | --- |
| 2969 | 2702 |

| first_person | non_first_person |
| --- | --- |
| 1617 | 873 |

| casual | looking | reflecting |
| --- | --- | --- |
| 707 | 516 | 393 |

@rumichunara

**Comment by rumichunara** [comment]
Hi Jason-
Thanks for these, all great. Working on integrating them into the text. I
know you are working on gender, and data from gnip, but just wanted to flag
that we might have to just fixup the figures a bit to make them readable in
the paper.
See attached paper template (2 columns, and how graphs will turn out).

I propose the following, since it might be too much to make ALL of the
figures 2 columns. Also can we have the figures in a high resolution format
(300 dpi)? PDF or JPEG or similar should work.

 Keep the following as 1 column figs and just increase size of labels:
-day vs probability
-wknd vs weekday (wave)

Use 2 columns for the following (to keep them readable, but also increase
label sizes):
-hour vs probability
-day of week classes
-hour of day classes
-hour and week classes

Will make 2 git requests following this.

You are also welcome to send me data for these plots and I can update them;
whichever is easiest for you.

Thanks!!
Rumi

**Comment by rumichunara** [comment]
added CSV file requests:

-The ROC plots

Thanks!
